template/page-xrp-overview.html.jinja

```diff
@@ -191,21 +191,19 @@
         <h5 class="longform mb-10">{% trans %}Exchanges are where people trade currencies. XRP is traded on more than 100 markets and exchanges worldwide.{% endtrans %}</h5>
         <p class="mb-10">{% trans %}There are different types of exchanges that vary depending on the type of market (spot, futures, options, swaps), and the type of security model (custodial, non-custodial).{% endtrans %}</p>
         <div class="card-grid card-grid-2xN mb-10">
-          <div class="mr-5-until-md">
+          <div>
```

Comment:
This and other cards in this section now have spacing between from the `gap` on the `card-grid-2xN` element that encompasses them.